src/octoprint/server/api/timelapse.py

```diff
@@ -113,6 +116,21 @@ def setTimelapseConfig():
 				else:
 					return make_response("Invalid value for interval: %d" % interval)
 
+		if "retractionZHop" in request.values:
+                        config["options"] = {
+                                "retractionZHop": 0
+                        }
+
+                        try:
+                                retractionZHop = float(request.values["retractionZHop"])
+                        except ValueError:
+                                return make_response("Invalid value for retraction Z-Hop: %r" % request.values["retractionZHop"])
+                        else:
+                                if retractionZHop > 0:
```

Comment:
IMHO there's no problem letting it be 0 and that should simply have been a `>=` instead of a `>`. Fixed accordingly on `maintenance`.